src/frontend/js/components/UserLogin/index.tsx

```diff
@@ -93,11 +93,13 @@ export const UserLogin = ({ profileUrls = [] }: UserLoginProps) => {
           user={{
             ...user,
             urls: [
-              ...profileUrls.map(({ label, action }) => ({
+              ...profileUrls.map(({ key, label, action }) => ({
```

Comment:
`profileUrls` is no more an array, this is an object.
To adapt the code to this new format, you could write:
```suggestion
              ...Object.entries(profileUrls).map(([key, { label, action }]) => ({
```

You also have to update Component Props type.

Reply:
:man_facepalming: